cmake/SetupCompilerFlags.cmake

```diff
@@ -21,3 +21,18 @@ endif ()
 if (ENABLE_HIP)
 	set(HIP_HIPCC_FLAGS "${HIP_HIPCC_FLAGS} -Wno-inconsistent-missing-override")
 endif()
+
+if (ENABLE_PEDANTIC_WARNINGS)
+  blt_append_custom_compiler_flag(
+    FLAGS_VAR UMPIRE_PEDANTIC_FLAG
+    DEFAULT  "-Wpedantic"
+    MSVC "/Wall /WX"
+    INTEL "-Wall -Wcheck -wd2259"
+  )
+
+  set(CMAKE_CXX_FLAGS "${UMPIRE_PEDANTIC_FLAG} ${CMAKE_CXX_FLAGS}")
```

Comment:
Do you still have to "add" the flag like this? I think you can use the blt macro to update `CMAKE_CXX_FLAGS` accordingly.

Reply:
Which blt macro do I use to add to `CMAKE_CXX_FLAGS`?  I looked in the documentation, but could not find anything (or I may have missed it).  Can you send a pointer to an example?

Reply:
I was mistaken. Modifying the variable directly is the correct way.